invenio/base/config.py

```diff
@@ -75,24 +75,29 @@
     'invenio.legacy.websubmit',
 ]
 
-CFG_PREFIX = distutils.sysconfig.get_config_var("prefix")
+_CFG_PREFIX = distutils.sysconfig.get_config_var("prefix")
```

Comment:
I'm not sure what is the best way to deprecate its usage, but there is an example:
```python
def cfg_prefix():
    from invenio.utils.deprecation import RemovedInInvenio22Warning
    warnings.warn('... CFG_PREFIX ...', RemovedInInvenio22Warning
    return distutils.sysconfig.get_config_var("prefix")

CFG_PREFIX = LocalProxy(cfg_prefix)
```